Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -1099,6 +1105,8 @@ RefPtr<Protocol::CSS::CSSRule> InspectorCSSAgent::buildObjectForRule(const Style
     if (!styleRule)
         return nullptr;
 
+    ASSERT(element.isConnected());
```

Comment:
NIT: Should this be at the top?  Or is there a scenario where `styleRule` is `nullptr` and the `element` is not attached?

Reply:
Yeah, it should probably be at the top to match our general style.